sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -0,0 +1,152 @@
+# Azure SDK Dynamic JSON samples
+
+Azure SDK client [protocol methods](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md) take `RequestContent` as an input parameter and return `Response` as their return type.  These types hold raw JSON content that can be accessed using Base Class Library (BCL) types such as `JsonDocument`, but use of these APIs can result in code that is difficult to read and obscures the author's intent.  To improve the developer experience, Azure.Core provides a [dynamic](https://learn.microsoft.com/dotnet/csharp/advanced-topics/interop/using-type-dynamic) layer over JSON APIs.
```

Comment:
Instead of saying "Base Class Library (BCL) types" I would say "any JSON parser, e.g. JsonDocument"

Reply:
Also, it's a bit inconsistent that you are taking about RequestContent. It's an input JsonDocument does not help with that. 

Lastly, the APIs are more general than protocol methods, and so maybe this section should start with "Some Azure SDK APIs return values that hold raw JSON, e.g. protocol methods.